packages/tao/src/commands/migrate.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('Migration', () => {
       expect(await migrator.updatePackageJson('mypackage', '2.0.0')).toEqual({
         migrations: [],
         packageJson: {
-          mypackage: { version: '2.0.0', alwaysAddToPackageJson: false },
+          mypackage: { version: '2.0.0', addToPackageJson: false },
```

Comment:
I think we need to keep the name `alwaysAddToPackageJson` since we have old migrations using that. Otherwise we'd need to support both `alwaysAddToPackageJson` _and_ `addToPackageJson`.

Reply:
`alwaysAddToPackageJson` is still supported in the migrations file, and should work the same as it does at the moment - but this PR essentially deprecates it, hence the test changes. but you raise a good point, i'll add a test that asserts that `alwaysAddToPackageJson` is still supported and works as expected 👍

Reply:
added a unit test, as well as the existing e2e test remains that tests `alwaysAddToPackageJson` 👍